openscenario/openscenario_interpreter/src/syntax/add_entity_action.cpp

```diff
@@ -79,11 +79,6 @@ try {
 
       TeleportAction::teleport(entity_ref, position);
       AssignControllerAction(entity.as<ScenarioObject>().object_controller)(entity_ref);
-      activatePerformanceAssertion(
-        entity_ref, vehicle.performance,
-        entity.as<ScenarioObject>().object_controller.is<Controller>()
-          ? entity.as<ScenarioObject>().object_controller.as<Controller>().properties
-          : Properties());
```

Comment:
Why do you completely remove this section?

Reply:
Because this method has become empty: [comment](https://github.com/tier4/scenario_simulator_v2/pull/979/files#r1146078172)